core/src/main/java/io/questdb/cairo/CairoConfiguration.java

```diff
@@ -643,6 +643,10 @@ default int getWalPurgeWaitBeforeDelete() {
 
     boolean isPartitionO3OverwriteControlEnabled();
 
+    default boolean isQueryTracingEnabled() {
```

Comment:
this has to be tested. The downside of having default methods is that there is a bunch of implementing classes that do not deal with this config. We also need reloadable property i guess?

Reply:
This was actually a stub, and a completely missing feature.

Resolved in 5a359727001c4ce11a705f204017f3123619229d.